### PR TITLE
refactor(components): [cascader,cascader-panel] add namespace

### PR DESCRIPTION
**Pull request by gjfei**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Comment by gjfei** [comment]
这个还需要我做什么？

**Comment by gjfei** [comment]
@sxzz 这个pr 我还需要做什么？

**Comment by sxzz** [comment]
@gjfei 抱歉，可能还需要改一些，还没进行 Review。

File: packages/components/cascader-panel/src/index.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
   <div
-    :class="['el-cascader-panel', border && 'is-bordered']"
+    :class="[ns.b('panel'), border && 'is-bordered']"
```
Comment (sxzz) [inline packages/components/cascader-panel/src/index.vue:3]:
```suggestion
    :class="[ns.b('panel'), ns.is('bordered', border)]"
```

File: packages/components/cascader-panel/src/index.vue [hunk 2]
```diff
@@ -273,13 +275,13 @@ export default defineComponent({
...
-          const container = (menuElement as HTMLElement).querySelector(
-            '.el-scrollbar__wrap'
+          const container = menuElement.querySelector(
+            `.${ns.namespace.value}scrollbar__wrap`
           )
           const activeNode =
-            menuElement.querySelector('.el-cascader-node.is-active') ||
-            menuElement.querySelector('.el-cascader-node.in-active-path')
-          scrollIntoView(container as HTMLElement, activeNode)
+            menuElement.querySelector(`.${ns.b('node')}.is-active`) ||
```
Comment (sxzz) [inline packages/components/cascader-panel/src/index.vue:282]: `is-active` same here

File: packages/components/cascader-panel/src/menu.vue [hunk 3]
```diff
@@ -3,9 +3,9 @@
     :key="menuId"
     tag="ul"
     role="menu"
-    class="el-cascader-menu"
-    wrap-class="el-cascader-menu__wrap"
-    :view-class="['el-cascader-menu__list', isEmpty && 'is-empty']"
+    :class="ns.b()"
+    :wrap-class="ns.e('wrap')"
+    :view-class="[ns.e('list'), isEmpty && 'is-empty']"
```
Comment (sxzz) [inline packages/components/cascader-panel/src/menu.vue:8]: same here

File: packages/components/cascader-panel/src/node.vue [hunk 4]
```diff
@@ -53,10 +50,10 @@
 
     <!-- postfix -->
     <template v-if="!isLeaf">
-      <el-icon v-if="node.loading" class="is-loading el-cascader-node__postfix">
+      <el-icon v-if="node.loading" :class="['is-loading', ns.e('postfix')]">
```
Comment (sxzz) [inline packages/components/cascader-panel/src/node.vue:53]: same here

File: packages/components/cascader/src/index.vue [hunk 5]
```diff
@@ -26,8 +26,8 @@
       <div
         v-clickoutside:[popperPaneRef]="() => togglePopperVisible(false)"
         :class="[
-          'el-cascader',
-          realSize && `el-cascader--${realSize}`,
+          ns.b(),
+          ns.m(realSize),
           { 'is-disabled': isDisabled },
```
Comment (sxzz) [inline packages/components/cascader/src/index.vue:31]: same here

File: packages/components/cascader/src/index.vue [hunk 6]
```diff
@@ -66,7 +66,7 @@
               v-else
               key="arrow-down"
               :class="[
-                'el-input__icon',
+                nsInput.e('icon'),
                 'icon-arrow-down',
                 popperVisible && 'is-reverse',
```
Comment (sxzz) [inline packages/components/cascader/src/index.vue:71]: same here

File: packages/components/cascader/src/index.vue [hunk 7]
```diff
@@ -124,18 +124,15 @@
...
       >
         <template v-if="suggestions.length">
           <li
             v-for="item in suggestions"
             :key="item.uid"
-            :class="[
-              'el-cascader__suggestion-item',
-              item.checked && 'is-checked',
-            ]"
+            :class="[ns.e('suggestion-item'), item.checked && 'is-checked']"
```
Comment (sxzz) [inline packages/components/cascader/src/index.vue:135]: same here

File: packages/components/cascader/src/index.vue [hunk 8]
```diff
@@ -322,6 +319,9 @@ export default defineComponent({
       COMPONENT_NAME,
       'popperAppendToBody'
     )
+    const ns = useNamespace('cascader')
```
Comment (sxzz) [inline packages/components/cascader/src/index.vue:322]:
```suggestion
    const nsCascader = useNamespace('cascader')
```

**Review by sxzz — COMMENTED** [review]
Please check if there are any missing `is-xxx`

**Comment by gjfei** [comment]
> Please check if there are any missing `is-xxx`

好的，我重新检查一下

**Review by sxzz — APPROVED** [review]

**Review by sxzz — APPROVED** [review]

**Pull request merged**